data/migrations/20190423082327_create_patients_table.js

```diff
@@ -0,0 +1,23 @@
+
+exports.up = function(knex, Promise) {
+  return knex.schema.createTable('patients', tbl => {
+      tbl.increments();
+      tbl.string('firstName', 255)
+        .notNullable();
+    tbl.string('lastName', 255)
+        .notNullable();
+    tbl.integer('age')
```

Comment:
Idea for the future: put a birthday here instead, and compute the age when needed